src/Pages/ADT3DScenePage/ADT3DScenePage.tsx

```diff
@@ -156,12 +154,6 @@ const ADT3DScenePageBase: React.FC<IADT3DScenePageProps> = ({
         },
         [deeplinkDispatch]
     );
-    const setBlobContainerUrl = useCallback(
```

Comment:
I see the blob setting was removed entirely. DOn't we need to still update the adapter somehwere?

Reply:
this was something you pointed in one of your earlier comment :) Since the adapter.setBlobContainerPath(url) is called with deeplink containerurl changes in useeffect now, no need to have a separate method for it.

Reply:
Gothca. I thought it was something that needed to move to a useEffect so was expecting a cut and paste and not jsut remove. If it works, then great!